docker-compose.client.yml

```diff
@@ -15,20 +15,32 @@ services:
       - BALENACLOUD_APP_NAME=${BALENACLOUD_APP_NAME}
     depends_on:
       - core
+    networks:
+      - levi
 
   core:
     privileged: true # preload requires docker-in-docker
     build: core
     volumes:
       - core-storage:/data
       - reports-storage:/reports
+      - /dev:/dev # required for creating losetup devices during preload
```

Comment:
This is almost certainly the wrong solution. Mapping `/dev` into a privileged container can cause permissions issues on the host, as the uids and gids don't match. You probably want to instead dynamically create the device nodes for `loop-control` and the loop devices at runtime, using the loop major and `loop-control` minor numbers, as done in leviathan-worker [here](https://github.com/balena-os/leviathan-worker/blob/a7335bde382d6fb1c6ce6765acb4feb2f8cea253/entry.sh#L32).

Reply:
^ Also not really relevant to this PR, as it didn't introduce this change

Reply:
Thanks @jakogut I will get this change plucked out into a different PR where we can continue on the discussion. For clarification, @alexgg did we even need to change this change? I reckon once we create our own network, it should be fine right?

Reply:
hey @vipulgupta2048 correct, there are two different problems. The change above is because the preloading fails in some systems (like mine) without it. I agree it's not the right change though.

Reply:
Cool beans, changes requested coming through.